subxt/src/tx/tx_client.rs

```diff
@@ -94,7 +94,7 @@ impl<T: Config, C: OfflineClientT<T>> TxClient<T, C> {
         params: <T::ExtrinsicParams as ExtrinsicParams<T>>::Params,
     ) -> Result<SubmittableExtrinsic<T, C>, Error>
     where
-        Call: PayloadT,
+        Call: Payload,
         Signer: SignerT<T>,
```

Comment:
I think this is fine, just thinking out loud, how would the following look?
- import `crate::tx::Signer` (not as `Signer T`)
- Use `<Call, Sign> where Sing: Signer<T>`?

But yea, we don't have to do it since we already have an `OfflineClientT and OnlineClientT` to distinguish between concret impl and traits

Reply:
Ohyeah, we do use the T pattern for `OfflineClient` and `OnlineClient` too don't we; I guess we are still not super consistent hehe. At least those traits are generally not supposed to be imported/used I guess!

> Use <Call, Sign> where Sing: Signer<T>?

I think that makes sense!